src/styles/_catalog.scss

```diff
@@ -1,22 +1,24 @@
 .catalog {
-  display: flex;
-  flex-wrap: wrap;
+  display: grid;
+  justify-content: center;
+  grid-template-columns: repeat(2, $cardsize);
+  gap: 48px;
 
-  max-width: 944px;
-  margin: 0 auto;
+  @media (min-width: 768px) {
```

Comment:
There should be one column below 488px
![image](https://user-images.githubusercontent.com/37161315/169849260-0944da08-adf3-44eb-a380-d4c2d8ab87dc.png)